cmd/kube-proxy/app/options/options.go

```diff
@@ -64,6 +65,7 @@ func NewProxyConfig() *ProxyServerConfig {
 		HealthzBindAddress:             net.ParseIP("127.0.0.1"),
 		OOMScoreAdj:                    qos.KubeProxyOOMScoreAdj,
 		ResourceContainer:              "/kube-proxy",
+		IptablesMasqueradeMark:         "0x4d410000/0xffff0000",
```

Comment:
why not just make it a single bit and just make the option be the bit number [0-31] ?